SPECS-EXTENDED/python-simpleline/python-simpleline.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "python-simpleline-1.6.tar.gz": "4288f61cc5f492d30aab64925bc60d9656bbf53600ef5892ec682dfe242d4382"
+  "python-simpleline-1.9.0.tar.gz": "7645e29a83df27a46defa1dc1e7e84f6adf91645e03cdd06b4384079bc89aef1"
```

Comment:
Since this tarball is already present in our internal blob store (azurelinuxsrcstorage) use its signature rather than this one.
Buddy build is failing because of this.